includes/core/class-user.php

```diff
@@ -1667,7 +1670,7 @@ function password_changed( $user_id = null ) {
 				um_fetch_user( $user_id );
 			}
 
-			UM()->mail()->send( um_user( 'user_email' ), 'changedpw_email' );
+			UM()->maybe_action_scheduler()->enqueue_async_action( 'um_send_deleted_user_email', array( um_user( 'user_email' ), 'changedpw_email' ) );
```

Comment:
probably you have to use `um_mas_send_email` here too

Reply:
These instances are fixed.